### PR TITLE
Functionality of dynamically updating the callflow, like branching.

### PR DESCRIPTION
**Pull request by aberke**
Does not add extra flow to Flows the way branching does.

### CODE REVIEW

File: applications/callflow/src/cf_exe.erl [hunk 1]
```diff
@@ -106,6 +106,13 @@ continue(Key, Call) ->
     Srv = whapps_call:kvs_fetch('consumer_pid', Call),
     continue(Key, Srv).
 
+-spec continue_with_flow(wh_json:object(), whapps_call:call() | pid()) -> 'ok'.
```
Comment (jamesaimonetti) [inline applications/callflow/src/cf_exe.erl]: This function is not exported and not used internally. I assume you meant to export it?

File: applications/callflow/src/cf_exe.erl [hunk 2]
```diff
@@ -353,6 +360,9 @@ handle_cast('transfer', State) ->
     {'stop', {'shutdown', 'transfer'}, State};
 handle_cast('control_usurped', State) ->
     {'stop', {'shutdown', 'control_usurped'}, State};
+handle_cast({'continue_with_flow', NewFlow}, #state{flow=Flow}=State) ->
```
Comment (jamesaimonetti) [inline applications/callflow/src/cf_exe.erl]: `Flow` is unused; either don't bind for it or prefix it with an '_'.

**Comment by aberke** [comment]
This was very sloppy.  I apologize.  I needed to put something down to come back to later.  Will fix now.

**Comment by aberke** [comment]
I amended the commit @jamesaimonetti

**Comment by jamesaimonetti** [comment]
Thanks @aberke, merged.

**Pull request merged**